opat/drupal-9.yaml

```diff
@@ -618,5 +619,133 @@ chapters:
             adherence:
               level: "partially-supports"
               notes: There is nothing to differentiate editing financial or legal data from any other data managed by Drupal.
+  success_criteria_level_aaa:
+    notes: Where possible the Drupal community strives to exceed AA compliance.
+    criteria:
+      - num: "1.2.6"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+      - num: "1.2.7"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+      - num: "1.2.8"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+      - num: "1.2.9"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+      - num: "1.4.6"
+        components:
+          - name: "web"
+            adherence:
+              level: "supports"
+              notes: A front-end and back-end theme could be configured to allow this to comply.
+      - num: "1.4.7"
+        components:
+          - name: "web"
+            adherence:
+              level: "not-applicable"
+              notes:
```

Comment:
@mgifford looks like we have empty `notes` here.

Reply:
Fixed.

Reply:
And yes, if you could at it to the catalog that would be great.